app/mailers/application_mailer.rb

```diff
@@ -0,0 +1,4 @@
+class ApplicationMailer < ActionMailer::Base
+  default from: 'from@example.com'
```

Comment:
@mcritchlow Yes, it looks like we can configure the [mailboxer.rb#17](https://github.com/ucsdlib/horton/blob/develop/config/initializers/mailboxer.rb#L7) for it. Would you like to use the dummy one (dams-support@ucsd.edu) from dams4 or just leave as-is for now?